src/lib/parser/YamlParser.ts

```diff
@@ -2,13 +2,12 @@
 import { HostParameterEntry } from "@microsoft/powerplatform-cli-wrapper/dist/host/IHostAbstractions";
 import fs = require('fs');
 import yaml = require('js-yaml');
-import path = require("path");
 
 export class YamlParser {
     public getHostParameterEntries(workingDir: string, actionFolder: string): Record<string, HostParameterEntry> {
         const parameterMap: Record<string, HostParameterEntry> = {};
         try {
-            const file = path.resolve(workingDir, `./${actionFolder}/action.yml`);
+            const file = workingDir + `/${actionFolder}/action.yml`;
```

Comment:
don't do path composition, that "cure" is usually making things worse... what is the actual issue you're fixing (that would be useful info in the change description, too). There's a corollary method, path.join() that does proper path construction as well

Reply:
Bug 2506010: [GitHub Actions] Path resolve in task parser fails on top of dist folder, this was the bug which was blocking me David, I filed it. After my PR went through, I updated the dist and made a commit, that seems to broke both the PR and rolling actions. Upon updating error log in yaml parser and generating and committing a dist again, I saw that path was not getting resolved properly.